.github/workflows/bulk-dep-upgrades.yaml

```diff
@@ -0,0 +1,59 @@
+name: Upgrade dependencies
+
+on:
+  schedule:
+    # Runs every 5 mins
+    - cron: '*/5 * * * *'
```

Comment:
This is just to test that this works. This workflow should not create another PR if one already exists. Once we verify this works we can change this to `0 0 1 * *` which will run  at 12:00AM on the first of every month